.gitignore

```diff
@@ -1,3 +1,4 @@
-vendor
-composer.lock
+.idea/
```

Comment:
Sure @stof , I just usually prefer insert these kind of line in the global `.gitignore` because they refer to the most spread editors.
Anyway it's ok if you prefer keep them out, I'll update the branch.